star-ros2/README.md

```
@@ -141,6 +141,98 @@ colcon test-result --verbose
+#   ros2 topic echo /odom/unfiltered
+#   ros2 topic echo /robot_status
+#   ros2 topic pub -r 100 /cmd_vel geometry_msgs/msg/Twist "linear: {x: 1.0}"
+./collect_baseline_metrics.sh 15 stress_test
+```
+
+### Prerequisites
+
+Before running baseline collection:
+
+1. **Build the Gateway binaries:**
+   ```bash
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)